manual/manual/style.css

```diff
@@ -55,13 +57,14 @@ div.h9 { background-color: #FFFFFF; }
 .paramstable { border-style : hidden ; padding: 5pt 5pt}
 body {
   background-color : #f7f7f7;
-  font-size: 17px;
-  padding-left: 5%;
-  padding-right: 5%;
+  font-size: 1rem;
+  max-width: 800px;
+  width: 85%;
+  margin: auto;
```

Comment:
What is the purpose of these changes? The `max-width` setting here is different from the one in `macros.hva`, there is a `width: 85%;`, and then `padding` turned into `margin: auto`, what are those doing?